src/runtime/entries/cloudflare.ts

```diff
@@ -60,3 +67,28 @@ const baseURLModifier = (request: Request) => {
 function normalizeOutgoingHeaders (headers: Record<string, string | string[] | undefined>) {
   return Object.entries(headers).map(([k, v]) => [k, Array.isArray(v) ? v.join(',') : v])
 }
+
+function toNodeListener (app: App, context: H3EventContext): NodeListener {
+  const toNodeHandle: NodeListener = async function (req, res) {
+    const event = createEvent(req, res)
+    event.context = { ...event.context, ...context }
```

Comment:
I think we might want to add CF context as `event.context.cloudflare` for better type support.

Reply:
```suggestion
    event.context = { ...event.context, cloudflare: context }
```

Reply:
Oops - didn't mean to edit. What about?

```suggestion
    event.context.cloudflare = context
```

Reply:
Thanks for feedback, implemented 🙌